cmake/O2Dependencies.cmake

```diff
@@ -162,6 +161,7 @@ o2_define_bucket(
 
     INCLUDE_DIRECTORIES
     ${FAIRROOT_INCLUDE_DIR}
+    ${FAIRROOT_INCLUDE_DIR}/fairmq # temporary fix, until bucket system works with imported targets
```

Comment:
Hi Dennis, 
It is very nice that FairRoot now exports targets. Indeed the bucket system does not handle it yet. I will create an issue for this. I might come back to you if I have questions. 
Cheers,
https://alice.its.cern.ch/jira/browse/OCCF-31